src/tdastro/astro_utils/passbands.py

```diff
@@ -0,0 +1,220 @@
+import os
+import time
+import urllib.request
+from pathlib import Path
+from urllib.error import HTTPError, URLError
+
+import numpy as np
+import scipy.integrate
+
+from tdastro.sources.physical_model import PhysicalModel
+
+
+class Passbands:
+    """Class to handle the transmission tables and normalization for different passbands."""
+
+    def __init__(self, bands=None):
```

Comment:
It would be nice to have an interface to manually set passbands. We can make a new issue to implement it later

Reply:
#54 - opened a new issue to keep track of this